integration-tests/deployment/keystone/ocr3_deployer.go

```diff
@@ -0,0 +1,38 @@
+package keystone
+
+import (
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/keystone/generated/ocr3_capability"
+)
+
+type OCR3Deployer struct {
+	lggr     logger.Logger
+	contract *ocr3_capability.OCR3Capability
+}
+
+func (c *OCR3Deployer) deploy(req deployRequest) (*deployResponse, error) {
+	ocr3Addr, tx, ocr3, err := ocr3_capability.DeployOCR3Capability(
+		req.Chain.DeployerKey,
+		req.Chain.Client)
+	if err != nil {
+		return nil, fmt.Errorf("failed to deploy OCR3Capability: %w", err)
+	}
+
+	_, err = req.Chain.Confirm(tx.Hash())
+	if err != nil {
+		return nil, fmt.Errorf("failed to confirm transaction %s: %w", tx.Hash(), err)
+	}
+	resp := &deployResponse{
+		Address: ocr3Addr,
+		Tx:      tx.Hash(),
+		Tv: deployment.TypeAndVersion{
+			Type:    OCR3Capability,
+			Version: deployment.Version1_0_0,
+		},
+	}
+	c.contract = ocr3
```

Comment:
Where does configuring OCR happen?

Reply:
same as above: thanks and see deploy.go ~ line 163